### ISSUE TITLE
Workloads in state shall not repeat the workload name as an attribute 

### ISSUE THREAD

**Issue by krucod3** [eclipse-ankaios/ankaios#185]
Workload name should be listed only once in the Ankaios state.

## Current Behavior
The Ankaios state currently lists workloads as:

```yaml
workloads: 
  hello2:
      name: hello2
      agent: agent_B
      tags:
      - key: owner
        value: Ankaios team
      dependencies: {}
      updateStrategy: AT_MOST_ONCE
      restart: true
      accessRights:
        allow: []
        deny: []
      runtime: podman
      runtimeConfig: |
        image: alpine:latest
        commandArgs: [ "echo", "Hello Ankaios"]
```

## Expected Behavior
The name is only specified as an object key:
```yaml
workloads: 
  hello2:
      agent: agent_B
      tags:
      - key: owner
        value: Ankaios team
      dependencies: {}
      updateStrategy: AT_MOST_ONCE
      restart: true
      accessRights:
        allow: []
        deny: []
      runtime: podman
      runtimeConfig: |
        image: alpine:latest
        commandArgs: [ "echo", "Hello Ankaios"]
```

## Steps to Reproduce

1. execute `ank get state`

## Context (Environment)

*

## Final result

To avoid the serialization we can just instrument serde with `skip_deserialization`
The input state is currently deserialized using a state parser, which is not used by the code in object.rs We should unify this and use a transfer object that does not have a name attribute and convert afterwards.

**To be filled by the one closing the issue.**

**Comment by krucod3** [comment]
I started some experiments in https://github.com/eclipse-ankaios/ankaios/tree/remove_wl_name_from_server_response to see what needs to be changed. The branch can be used as a base for the changes.

**Comment by krucod3** [comment]
We needed to do some changes on the internal workload representation and as a side effect the `name` was removed from the external state representation.
The issue here will be fixed by #189.

**Comment by krucod3** [comment]
This is now fixed.
